components/text_normalization/tests/component_test.py

```diff
@@ -0,0 +1,50 @@
+import json
+import os
+import typing as t
+from glob import glob
+
+import pandas as pd
+import pytest
+from fondant.component_spec import ComponentSpec
+
+from components.text_normalization.src.main import TextNormalizationComponent
```

Comment:
Why do we import this from the repository root? I would expect us to be working inside the component folder here. So `from src.main import TextNormalizationComponent`.

We'll have to make sure this still works when running as part of the docker build as you proposed in https://github.com/ml6team/fondant/issues/301 though.